src/Sylius/Bundle/ApiBundle/Resources/config/app/config.yaml

```diff
@@ -10,6 +10,7 @@ parameters:
     sylius.api.doctrine_extension.order_visitor_item.filter_cart.allowed_non_get_operations:
         - "shop_select_payment_method"
     sylius.api.paths_to_hide:
+        - "%sylius.security.new_api_route%/admin/address-log-entry/{id}"
```

Comment:
I'm afraid it does not work currently on APIP 3 branch if I see correctly

Reply:
Yes, but I have added it so we wouldn't forget about it.